src/api/spec/support/shared_examples/features/bootstrap_user_tab.rb

```diff
@@ -41,9 +41,8 @@
     end
 
     it 'Add non existent user' do
-      add_user_link = page.find('a', text: 'Add User')
-      page.scroll_to(add_user_link)
-      add_user_link.click
+      # HACK: on mobile, when `screen-offset-top` CSS rule is in place, scrolling and clicking at capybara level does not work
```

Comment:
@ncounter tiny thing, but I would suggest to use the term `FIXME:` instead or in addition to `HACK:`. This way its easier to grep through the code in the future to possibly find things to fix :D